### PR TITLE
Update autocomplete.html

### PR DESCRIPTION
**Pull request by ItWorksOnMyMachine**
In relation to issue #37.

### CODE REVIEW

**Comment by ItWorksOnMyMachine** [comment]
the lastFindPromise is defined on the view model only after the first call to valueChanged, which doesn't happen until after the first setFocus. This causes the "No results" list item to show because the truthy type coercion of undefined results in false and !lastFindPromise ends up being true.

File: src/component/bootstrap/autocomplete.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
       <li if.bind="lastFindPromise && results.length" role="separator" class="divider"></li>
 
       <li
-        show.bind="results.length === 0 && !lastFindPromise"
+        show.bind="results.length === 0 && lastFindPromise != undefined && !lastFindPromise"
```
Comment (RWOverdijk) [inline src/component/bootstrap/autocomplete.html]: Please use !==

**Comment by RWOverdijk** [comment]
Cool nick!

One question, could you use our commit message format? https://github.com/SpoonX/about/blob/master/CONTRIBUTING.md#commit

**Comment by ItWorksOnMyMachine** [comment]
Sorry about the comment, this is my first pull request and I totally missed the requirement.  I've updated the comment and changed the non-equality to a strict non-equality for consistency.

**Pull request closed without merging**